includes/Channel.hpp

```diff
@@ -13,40 +13,31 @@
 #ifndef INCLUDES_CHANNEL_HPP_
 #define INCLUDES_CHANNEL_HPP_
 
-#include <arpa/inet.h>
-#include <fcntl.h>
-#include <netinet/in.h>
-#include <poll.h>
-#include <sys/socket.h>
-#include <sys/types.h>
-#include <unistd.h>
-
-#include <csignal>
-#include <cstdlib>
-#include <cstring>
-#include <iostream>
-#include <vector>
 #include <map>
-#include <algorithm>
+#include <string>
 
-#include "../includes/Client.hpp"
+#include "Client.hpp"
 
-class Channel {
- private:
-  std::string                _name;
-  std::string                _topic;
-  std::map<int, Client>      _clients;
+class Client;
+
+typedef std::map<int, Client> clientsMap;
+typedef Client &ClientRef;
 
+class Channel {
+private:
```

Comment:
includes/Channel.hpp:27:  private: should be indented +1 space inside class Channel  [whitespace/indent] [3]